### DISCUSSION TITLE
Time limit option for commoncrawl source

### DISCUSSION THREAD

**Discussion by MirageTurtle** [projectdiscovery/subfinder#488] [Q&A] [answered]
Does subfinder support the feature of time limit? The time limit means the record time. For example, I want to get some subdomains from common crawl in November.

By the way, Which time section does subfinder choose? Or its time rule?

I'll appreciate it if you answer this question.

**Comment by ehsandeep** [comment, marked as answer]
@MirageTurtle we can not specify the time, time information is hardcoded here - https://github.com/projectdiscovery/subfinder/blob/master/v2/pkg/subscraping/sources/commoncrawl/commoncrawl.go#L25.

**Reply by MirageTurtle** [reply]
Thanks for answer!
